README.md

```diff
@@ -1,24 +1,70 @@
 # VirtoCommerce.Inventory
-VirtoCommerce.Inventory module represents a store inventory management system.
 
-![Inventory management UI](https://cloud.githubusercontent.com/assets/5801549/15571240/01a76534-233a-11e6-8cd1-1281c7de22f9.png)
+## Overview
 
-# Documentation
-User guide: <a href="https://virtocommerce.com/docs/vc2userguide/fulfillment" target="_blank">Fulfillment</a>
+Inventory management is a system of stock level controlling and fullfilment centers management. 
 
+Inventory is often the largest item a business has in its current assets, meaning it must be accurately monitored. Inventory is counted and valued at the end of each accounting period to determine the company's profits or losses.
+
+In the ecommerce sector, a day-to-day management of inventory helps increase business intelligence and visibility. Ecommerce business owners have better ongoing control when they have constant eyes on inventory.
+
+## VirtoCommenrce Inventory Module Description
+
+VirtoCommerce.Inventory module represents a store inventory management system. The main goal of the Inventory module is stock level controlling and managing of fulfillment centers.
+
+The VC Inventory module is customizable and supports metafields.
+
+The Inventory module content can be both exported and imported using the VC Export/Import functionality.
+
+![Inventory module](docs/media/screen-inventory.png)
+
+Fields description :
+
+1. 'In stock' - total quantity of product items allowed in stock;
+1. 'Reserved'- in VC this field has no logics and is simply a UI element;
```

Comment:
has no logic?

Reply:
Logics! Yes, means it's simply a UI element. May be I should rephrase it ?

Reply:
Rephrase it please